www/src/components/ReactPlayground.js

```diff
@@ -114,172 +117,168 @@ const StyledExample = styled('div')`
 
 let uid = 0;
 
-class Editor extends React.Component {
-  state = { ignoreTab: false };
-
-  id = `described-by-${++uid}`;
-
-  handleKeyDown = event => {
-    const { key } = event;
-
-    if (this.state.ignoreTab && key !== 'Tab' && key !== 'Shift') {
-      if (key === 'Enter') event.preventDefault();
-      this.setState({ ignoreTab: false });
-    }
-    if (!this.state.ignoreTab && key === 'Escape') {
-      this.setState({ ignoreTab: true });
-    }
-  };
-
-  handleFocus = e => {
-    if (e.target !== e.currentTarget) return;
```

Comment:
I think the same styling changes that broke the CSS also make this check no-longer-necessary.